### PR TITLE
[DOCKER] Configurable NDK version support

### PR DESCRIPTION
**Pull request by srkreddy1238**
Let the Android NDK version configurable as a command line argument

### CODE REVIEW

File: docker/Dockerfile.ci_adreno [hunk 1]
```diff
@@ -22,10 +22,10 @@ COPY utils/apt-install-and-clear.sh /usr/local/bin/apt-install-and-clear
 
 # Android SDK
 COPY install/ubuntu_install_androidsdk.sh /install/ubuntu_install_androidsdk.sh
-RUN bash /install/ubuntu_install_androidsdk.sh
+RUN bash /install/ubuntu_install_androidsdk.sh 25.2.9519653 25
 ENV ANDROID_HOME=/opt/android-sdk-linux
-ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147
-ENV ANDROID_NDK_MAJOR=21
+ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/25.2.9519653
+ENV ANDROID_NDK_MAJOR=25
```
Comment (echuraev) [inline docker/Dockerfile.ci_adreno:28]: Is it necessary to specify these variables here? I see that they will be added to `/etc/profile` in `ubuntu_install_androidsdk.sh`.

File: docker/Dockerfile.ci_adreno [hunk 2]
```diff
@@ -22,10 +22,7 @@ COPY utils/apt-install-and-clear.sh /usr/local/bin/apt-install-and-clear
 
 # Android SDK
 COPY install/ubuntu_install_androidsdk.sh /install/ubuntu_install_androidsdk.sh
-RUN bash /install/ubuntu_install_androidsdk.sh
-ENV ANDROID_HOME=/opt/android-sdk-linux
-ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147
-ENV ANDROID_NDK_MAJOR=21
+RUN bash /install/ubuntu_install_androidsdk.sh 25.2.9519653 25
```
Comment (echuraev) [inline docker/Dockerfile.ci_adreno:25]: Sorry, one more comment. Probably we shouldn't pass `ANDROID_NDK_MAJOR`? We can parse this value from `ANDROID_NDK_VERSION`.

Comment (srkreddy1238) [reply docker/Dockerfile.ci_adreno]: Done. Made few other tools versions too configurable. Cmake was causing some warnings over older version with new NDK.

**Review by echuraev — APPROVED** [review]
Sorry for the delay, I was on vacation. LGTM

**Pull request merged**